setup.py

```diff
@@ -52,7 +52,6 @@
         'future>0.14.0',
         'lzstring',
         'jinja2>=2.9',
-        'matplotlib',
```

Comment:
Oh, my bad. I had this removed in my fork because of some issues installing matplotlib. Sorry about that Phil, I'll put it back.